editor/editor_node.cpp

```diff
@@ -3692,7 +3692,7 @@ void EditorNode::_set_current_scene_nocheck(int p_idx) {
 	_edit_current(true);
 
 	_update_title();
-	scene_tabs->update_scene_tabs();
+	scene_tabs->call_deferred(SNAME("update_scene_tabs"));
```

Comment:
```suggestion
	callable_mp(scene_tabs, &EditorSceneTabs::update_scene_tabs).call_deferred();
```
This method isn't exposed as far as I can see, so can't be called like this, and this is the preferred method in any case